app/views/service_providers/show.html.erb

```diff
@@ -90,10 +90,9 @@
 <h2><label for="push_notification_url">Push Notification URL:</label></h2>
 <p class="font-mono-xs margin-top-0" name="push_notification_url"><%= service_provider.push_notification_url %></p>
 
-    <% if service_provider.identity_protocol == 'openid_connect' %>
+    <% if service_provider.oidc? %>
```

Comment:
I suggest we get rid of this condition and call this "Additional Redirect URIs" for OIDC and SAML

Reply:
I don't mind keeping this for now, I think the language is clearer and this whole page will eventually get refactored so we can improve it then. Thanks though!